python/graphstorm/dataloading/dataset.py

```diff
@@ -416,6 +416,44 @@ def eval_etypes(self):
         """edge type for evaluation"""
         return self._eval_etypes
 
+class GSgnnLPTrainData(GSgnnEdgeTrainData):
+    """ Link prediction training data
+
+    Parameters
+    ----------
+    graph_name : str
+        The graph name
+    part_config : str
+        The path of the partition configuration file.
+    train_etypes : tuple of str or list of tuples
+        Target edge types for training
+    eval_etypes : tuple of str or list of tuples
+        Target edge types for evaluation
+    label_field : str
+        The field for storing labels
+    node_feat_field: str or dict of str
```

Comment:
do we support multiple feature fields for one node type? the same question applies to edge features and positive graph features.

Reply:
We do. Actually node_feat_field should be str or dist of list of str.

Reply:
I will update the docstring.